libs/execution_context/ebpf_core.c

```diff
@@ -12,6 +12,9 @@
 #include "ebpf_serialize.h"
 #include "ebpf_state.h"
 
+extern ebpf_helper_function_prototype_t* _ebpf_core_helper_function_prototype;
+extern uint32_t _ebpf_core_helper_functions_count;
```

Comment:
functions starting with `_` should be static per point 7 of https://github.com/microsoft/ebpf-for-windows/blob/master/docs/DevelopmentGuide.md#naming-conventions.  Non-static functions should not start with `_`